drones/connection/connector_template.py

```diff
@@ -0,0 +1,68 @@
+"""Tello drone connection and communication.
+
+Additional info.
+"""
+
+import logging
+import threading
+from typing import List
+
+from ..common.movement_instruction import MovementInstruction
+from ..common.drone_state import DroneState
+
+log = logging.getLogger()
+
+
+class ConnectorTemplate:
+    """Tello drone connection manager
+    TODO:
+    """
+
+    _response_thread: threading.Thread
```

Comment:
Are you sure you want to make those attributes as class attributes? As I can see, they should be instance attributes i.e. moved to `__init__`